### PR TITLE
[Healthcareapis] Fixing bugs with managed identity and export storage

### PR DESCRIPTION
**Pull request by iviark**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [X] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [X] Have you run `python scripts/ci/test_index.py -q` locally?

For new extensions:

- [X] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your PR is merged into master branch, a new PR will be created to update `src/index.json` automatically.  
The precondition is to put your code inside this repo and upgrade the version in the PR but do not modify `src/index.json`.

### CODE REVIEW

**Review by feordin — APPROVED** [review]

**Comment by yonzhan** [comment]
Healthcareapis

File: src/healthcareapis/azext_healthcareapis/generated/custom.py [hunk 1]
```diff
@@ -43,14 +43,24 @@ def healthcareapis_service_create(client,
...
+
     properties = {
         'access_policies': access_policies,
         'cosmos_db_configuration': cosmos_db_configuration,
         'authentication_configuration': authentication_configuration,
         'cors_configuration': cors_configuration,
         'private_endpoint_connections': private_endpoint_connections,
         'public_network_access': public_network_access,
-        'export_configuration_storage_account_name': export_configuration_storage_account_name
+        'export_configuration': export_configuration
```
Comment (Juliehzl) [inline src/healthcareapis/azext_healthcareapis/generated/custom.py:63]: all these code are in generated folder. If you want to manually override it, please add change in manual folder.

File: src/healthcareapis/azext_healthcareapis/manual/custom.py [hunk 2]
```diff
@@ -0,0 +1,262 @@
...
+        'name': resource_name,
+        'kind': kind,
+        'location': location,
+        'tags': tags,
+        'etag': etag,
+        'identity': service_identity_type,
+        'properties': properties
+    }
+
+    print(service_description)
```
Comment (fengzhou-msft) [inline src/healthcareapis/azext_healthcareapis/manual/custom.py:76]: Is this line for debug purpose?

File: src/healthcareapis/azext_healthcareapis/manual/custom.py [hunk 3]
```diff
@@ -0,0 +1,262 @@
...
+                                  tags=None,
+                                  public_network_access=None,
+                                  no_wait=False):
+
+    return sdk_no_wait(no_wait,
+                       client.update,
+                       resource_group_name=resource_group_name,
+                       resource_name=resource_name,
+                       tags=tags,
+                       public_network_access=public_network_access)
```
Comment (fengzhou-msft) [inline src/healthcareapis/azext_healthcareapis/manual/custom.py:97]: Is this function directly copied from the generated `custom.py`? You only need to put the functions that you need to modify in this manual `custom.py`. The 2 files will be merged and the function in the manual folder will overwrite the same-name function in generated folder.

File: src/healthcareapis/azext_healthcareapis/manual/custom.py (hunk 2 above)
Comment (iviark) [reply src/healthcareapis/azext_healthcareapis/manual/custom.py]: It was for debugging, removed.

File: src/healthcareapis/azext_healthcareapis/manual/custom.py (hunk 3 above)
Comment (iviark) [reply src/healthcareapis/azext_healthcareapis/manual/custom.py]: Removed all methods that weren't changed

**Comment by fengzhou-msft** [comment]
Please also update version in `setup.py`.

**Pull request merged**